pydiggy/node.py

```diff
@@ -136,24 +123,42 @@ def __new__(cls, name, bases, attrs, **kwargs):
 
 
 class Node(metaclass=NodeMeta):
-    uid: int

```

Comment:
Why remove this? It will be missing from the `annotations` and not added as a property, unless it is somewhere else now?

Reply:
I just removed the class variable. The instance variable self.uid is still declared in __init__, and therefore all of the instances still have it. I removed it because I thought it was confusing, and it was never actually used.

Reply:
I think it should stay so that the `uid` is declared as a type annotation.

Reply:
Okay. I put it back, and added a comment so that future me will not be confused (if that's okay).